src/apps/seo/src/store/redirects.js

```diff
@@ -68,13 +68,20 @@ export function createRedirect(redirect) {
     dispatch({
       type: "REDIRECT_CREATE"
     });
-    request(`${CONFIG.API_INSTANCE}/web/redirects`, {
+    return request(`${CONFIG.API_INSTANCE}/web/redirects`, {
       method: "POST",
       json: "true",
       body: redirect
     })
       .then(json => {
         if (!json.error) {
+          dispatch(
+            notify({
+              kind: "success",
+              message: `Created new redirect`
```

Comment:
We should drop "new" as it's superfluous. "Created" implies it's new.

Reply:
For the errors later we should match the creation structure. e.g.
Change: "Failed to create SEO redirect"
To: "Failed creating redirect"